### PR TITLE
Collect fs files to git repo

### PR DESCRIPTION
**Pull request by humhei**
PR for https://github.com/fable-compiler/ts2fable/issues/165
It worked 
Take a Look at
https://github.com/fable-compiler/ts2fable-exports/commits/dev
### Commit strategy
- When PR is merged. Commits will be added to [master branch](https://github.com/fable-compiler/ts2fable-exports/tree/master)
- When PR is not merged. Commits will be added to [dev branch](https://github.com/fable-compiler/ts2fable-exports/tree/dev)

> It'll be also very good if we could annotate the bindings with the version of the TS declaration file used.

I will add this feature and README structure like below in future

| ts2fable compiled file  | npm package| npm package version |
| :-------------: | :-------------: | :-------------: |
| [React.fs](https://github.com/humhei/ts2fable-exports/React.fs)  |  [react](https://www.npmjs.com/package/react) | 16.2.0

### CODE REVIEW

File: tools/deploy.fsx [hunk 1]
```diff
@@ -101,6 +116,64 @@ Target.Create "RunTest" (fun _ ->
...
+                else "-b dev git@github.com:fable-compiler/ts2fable-exports.git"
+
+            let repositoryDir = "./ts2fable-exports"
+            Shell.DeleteDir repositoryDir
+            clone "./" sshUrl repositoryDir
+            CopyDir repositoryDir testCompileDir (fun f -> f.EndsWith ".fs")
+            StageAll repositoryDir
+            try 
+                let descripton = 
+                    [ sprintf "Appveryor https://ci.appveyor.com/project/fable-compiler/ts2fable/build/%s" buildVersion
```
Comment (alfonsogarciacaro) [inline tools/deploy.fsx]: Minor typo: `Appveyor`

**Comment by alfonsogarciacaro** [comment]
This is awesome, @humhei, thank you! :clap: :tada: :clap:

Now we only need a way so users can easily find which bindings are available and download them :smile:

**Comment by StachuDotNet** [comment]
This is really cool. 

I've just bought definitelyfable.org on a whim (who doesn't collect domain names?)
Let's do something with this, or would it be better to have these listed on the fable.io website?

**Comment by humhei** [comment]
@StachuDotNet 

> I've just bought definitelyfable.org on a whim (who doesn't collect domain names?)

😃 
Thanks very much for doing this. 
I think we should come back to interact with it some day

> have users find fable bindings more easily

I use this https://github.com/fable-compiler/ts2fable/issues/165 to track these features

**Pull request merged**